include/libtorrent/kademlia/get_item.hpp

```diff
@@ -53,13 +53,15 @@ class get_item : public find_data
 	// for immutable itms
 	get_item(node& dht_node
 		, node_id target
-		, data_callback const& dcallback);
+		, data_callback const& dcallback
+		, nodes_callback const& ncallback);
```

Comment:
allright, nodes_callback seems to be called when a find_data traversal completes. and it passes on the closest DHT nodes we found during the search, the ones we may want to put to.